conf/janus.plugin.videoroom.cfg.sample

```diff
@@ -10,6 +10,9 @@
 ; audiocodec = opus|isac32|isac16|pcmu|pcma (audio codec to force on publishers, default=opus)
 ; videocodec = vp8|vp9|h264 (video codec to force on publishers, default=vp8)
 ; audiolevel_ext = yes|no (whether the ssrc-audio-level RTP extension must
+; audiolevel_event = yes|no (whether to emit event to other users or not, default=no)
+; audio_active_packets = 100 (number of packets with audio level, default=100, 2 seconds)
+; audio_level_average = 25 (average value of audio level, 127=muted, 0='too loud', default=25)
 ;		be negotiated/used or not for new publishers, default=yes)
```

Comment:
Looks like the `be negotiated` part belongs to after `audiolevel_ext = yes|no`, possibly a copy/paste error when you added your new settings?